atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -196,7 +214,8 @@ def _get_pullspecs(self, operator_csv):
                    "automatically.".format(operator_csv.path))
             raise RuntimeError(msg)
         else:
-            self.log.warning("%s has a relatedImages section, skipping", operator_csv.path)
+            related_images = operator_csv._related_image_pullspecs()
```

Comment:
This method is internal to the operator module, as is the return value (list of `NamedPullspec` objects). Should the there be a public `get_related_image_pullspecs()` method which returns a set of `ImageName` objects?

Reply:
Thanks @chmeliik. Updated.

Reply:
IMO the public method should return a collection of `ImageName`, not `NamedPullspec`. `NamedPullspec` was intended to be a private part of the operator module. I think the `_related_image_pullspecs` method should be kept and `get_related_image_pullspecs` should be a new method.

Reply:
@chmeliik PTAL

Reply:
Looks good :+1: